recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+    - msvc_fixes.diff
+
+build:
+  number: 0
+  skip: True        # [np!=112 or (win and py27)]
+  features:
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - gcc                   # [linux]
+    - cmake
+    - eigen
+    - numpy x.x
+    - itk       4.11.*
+    - boost     1.63.*
+    - scikit-learn
```

Comment:
Is this only a build requirement?

Reply:
Tests should be run in the test section, not in the build script :)

Reply:
This raises a question that (I think) there isn't any good answer to.  Many C++ libraries include tests that can be executed via `make test` or `make check`, etc.  But those tests assume the binary *hasn't been installed yet*.  They test it in-place.  Conda-build's `test` step happens too late to run those checks.  What is considered "best practice" in such cases?  For now, I think simply running the tests as part of the `build` step is reasonable, but I'm open to suggestions.

Side note: It would be nice of `conda-build` exposed an environment variable (say, `NO_TEST`) within `build.sh` to indicate whether or not the user is building the recipe with the `--no-test` flag...

Reply:
Yes, that is true @stuarteberg - I even maintain a bunch like that.  But this package isn't in that situation, is my understanding :)